.github/workflows/dev_on_workflow_update_api_client_template.yml

```diff
@@ -48,33 +63,61 @@ jobs:
           else
             echo "should-run-cache=false" | tee -a $GITHUB_OUTPUT
           fi
-      - uses: actions/setup-node@v2
-        if: steps.evaluate-variables.outputs.should-run-cache == 'true'
-        with:
-          node-version-file: "pro/.nvmrc"
-      - name: "Install dockerize"
-        if: steps.evaluate-variables.outputs.should-run-cache == 'true'
-        run: ./scripts/install_dockerize.sh $DOCKERIZE_VERSION
-      - name: "Cache"
-        if: steps.evaluate-variables.outputs.should-run-cache == 'true'
-        uses: satackey/action-docker-layer-caching@v0.0.11
-        # Ignore the failure of a step and avoid terminating the job.
-        continue-on-error: true
-        with:
-          key: docker-cache-${{ hashFiles('api/poetry.lock') }}
-          restore-keys: |
-            docker-cache-${{ hashFiles('api/poetry.lock') }}
-            docker-cache
       - name: "Fix local permissions"
         if: steps.evaluate-variables.outputs.should-run-cache == 'true'
         run: sudo chown -R $PCAPI_UID:$PCAPI_GID .
         working-directory: api
         env:
           PCAPI_UID: 1000
           PCAPI_GID: 1000
+      - uses: KengoTODA/actions-setup-docker-compose@v1
```

Comment:
Ci qui fail, le `if: steps.evaluate-variables.outputs.should-run-cache == 'true'` semble nécessaire